common/app/implicits/Requests.scala

```diff
@@ -66,6 +71,8 @@ trait Requests {
 
     lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || (!r.host.isEmpty && r.host == Configuration.amp.host)
 
+    lazy val isApps: Boolean = r.getQueryString("apps").isDefined
```

Comment:
We might want to have a special domain for apps in the future such as `m.guardianapis.co.uk`, but I think for the time being we can make do with just a query string. This is easy enough to change if we need to.